docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM mrcide/naomi-base:latest
+FROM mrcide/naomi-base:add-latex-lib
```

Comment:
I need to unpin this after https://github.com/mrc-ide/naomi.docker/pull/8 has been merged